package.json

```diff
@@ -68,8 +68,6 @@
     "copy-to-clipboard": "^3.0.8",
     "country-flag-icons": "^1.4.19",
```

Comment:
While we're removing things, I don't think this is used anymore

Reply:
Shouldn't affect bundle size, but it's been removed.